.github/workflows/reusable-build.yml

```diff
@@ -0,0 +1,97 @@
+name: Reusable build steps
+
+on:
+  workflow_call:
+
+jobs:
+  setup-java-and-cache:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Set up JDK for x64
+      uses: actions/setup-java@v4
+      with:
+        java-version: '11'
```

Comment:
Hier wird es über kurz oder lang zu Problemen kommen, sobald was nächste offizielle Release von Jameica oder Hibiscus mit Java > 11 erzeugt wird. Darauf hatte ich schon in https://github.com/openjverein/jverein/issues/516#issuecomment-2520195773 hingewiesen.

Reply:
Ich sehe auch keinen Grund, es nicht jetzt schon zu ändern. Nennt sich forward compatibility. 😏
Ist aber auch egal.